specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2020-08-06-preview/baremetalinfrastructure.json

```diff
@@ -0,0 +1,877 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-08-06-preview",
+    "title": "baremetalinfrastructure",
+    "description": "Description of the new service",
```

Comment:
Copy pasta? I think we need a specific description

Reply:
that was a part of the template that was auto-generated. thanks will update to describe our service

Reply:
It has been updated